src/rich-text-editor.component.js

```diff
@@ -52,13 +52,19 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
 
   function emptyEditor() {
     // do it with selection and execCommand so it can be undone with Ctrl Z
-    const range = document.createRange()
-    range.selectNodeContents(divRef.current)
+    setSelection()
+    document.execCommand('removeFormat')
+    document.execCommand('delete')
+  }
+
+  function setSelection(collapse) {
+    const range = document.createRange() // start and end of nodes
+    range.selectNodeContents(divRef.current) // setting the start and end nodes
+    collapse && range.collapse(false) // collapsed range starts and ends on the same point (the end for focus)
```

Comment:
Expressions have return values, blocks do not. It's a bit weird to use an expression instead of a block when you don't care about the return value:

```suggestion
if (collapse) range.collapse(false) // collapsed range starts and ends on the same point (the end for focus)
```